gum/backend-x86/gumstalker-x86.c

```diff
@@ -3470,6 +3458,91 @@ gum_exec_block_write_call_probe_code (GumExecBlock * block,
   }
 }
 
+#if GLIB_SIZEOF_VOID_P == 4 && !defined (HAVE_QNX)
+static void
+gum_exec_block_write_sysenter_continuation_code (GumExecBlock * block,
+                                     GumGeneratorContext * gc,
+                                     gpointer saved_ret_addr)
```

Comment:
These should align with the opening parenthesis, and if any line overflows 80 cols, all should be hanging indents (4 spaces).